packages/ember-htmlbars/tests/node-managers/view-node-manager-test.js

```diff
@@ -0,0 +1,63 @@
+import ViewNodeManager from 'ember-htmlbars/node-managers/view-node-manager';
```

Comment:
These tests are to confirm the [changes in the assertion](https://github.com/emberjs/ember.js/pull/12370/files#r40686699) work properly (that is why there isn't a corresponding change to `component-node-manager.js`).